specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -410,6 +410,43 @@
         "deprecated": false
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.IoTCentral/appTemplates": {
```

Comment:
>appTemplates [](start = 68, length = 12)

You're also missing a route for getting a single appTemplate resource. That's required per the RPC ([https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource))

Reply:
sorry @majastrz , the new updated PR is here: https://github.com/Azure/azure-rest-api-specs/pull/5495, sorry for the confusion, please continue taking a look there, I will close this one.Thanks

Reply:
No worries. I added a "do not merge" label just in case.